app/models/architecture.rb

```diff
@@ -4,6 +4,8 @@ class Architecture < ActiveRecord::Base
   friendly_id :name
   include Parameterizable::ByIdName
 
+  attr_accessible :name, :operatingsystem_ids
```

Comment:
Can you create a refactor ticket to take advantage of the new in-controller strong params, if we don't have that?